crates/orderbook/openapi.yml

```diff
@@ -917,8 +923,7 @@ components:
           description: Quote a sell order given the final total sell amount including fees
           properties:
             kind:
-              type: string
-              enum: [sell]
+              $ref: "#/components/schemas/OrderKind"
```

Comment:
This is intentionally `[sell]`, as the use with `oneOf` is roughly equivalent to TS:

```ts
type OrderQuoteSide =
    | { kind: "sell", sellAmountBeforeFee: TokenAmount }
    | { kind: "sell", sellAmountAfterFee: TokenAmount }
    | { kind: "buy", buyAmountAfterFee: TokenAmount };
```

Reply:
Oops, sorry. It makes some problems with typescript code based on the scheme, but it's our problems :) 
Fixed